src/pages/myAdmin/guides/codeBase/usingWithJavascript.tsx

```diff
@@ -93,35 +93,41 @@ myAdminApi().call('GetDevicePlans', devicePlanParams, function(devicePlans) {
 
 const moreInformation: ReactNode = (
   <div className="paragraph">
-  <p>For more information, see the <a href="../../code-samples/javascript-examples">{/*TODO: fix link */}JavaScript Examples</a> section.</p>
+    <p>
+      For more information, see the{" "}
+      <Link to="/myAdmin/codeSamples/javascriptExamples">
+        JavaScript Examples
+      </Link>{" "}
+      section.
+    </p>
   </div>
 );
 
 const pageTitle: PageTitleProps = {
   title: "Using With JavaScript",
-  breadCrumbItems: ["MYA", "Guides", "Code Base", "Using With JavaScript"]
+  breadCrumbItems: ["MYA", "Guides", "Code Base", "Using With JavaScript"],
```

Comment:
Just noting that trailing comma was added here, but once the ESLint and prettier config is set up, it should automatically be formatted to remove all trailing commas